langport/model/model_args.py

```diff
@@ -28,6 +28,12 @@ def add_model_args(parser):
     parser.add_argument(
```

Comment:
Should we move these args to somewhere? Because they are only valid for the huggingface worker.